From 57d38867035ee10b33c1dd846982312da3e01db8 Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Wed, 5 Feb 2020 01:46:42 +0530
Subject: [PATCH 5/5] Enable more gradle optimizations

Nope, never use daemon, we're just "compile and use" use-case
---
 gradle.properties | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/gradle.properties b/gradle.properties
index 7198fa92..56929034 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -1,4 +1,6 @@
-org.gradle.daemon=true
+org.gradle.daemon=false
 org.gradle.jvmargs=-Xmx2560m
 android.enableJetifier=true
-android.useAndroidX=true
\ No newline at end of file
+android.useAndroidX=true
+org.gradle.parallel=true
+android.enableD8=true
\ No newline at end of file
-- 
2.17.0.windows.1

